subsys/dfu/dfu_target/Kconfig

```diff
@@ -121,10 +121,54 @@ config DFU_TARGET_SUIT
 	select SUIT_UTILS
 	select ZCBOR
 	select ZCBOR_CANONICAL
+	select FLASH
+	select STREAM_FLASH
+	select STREAM_FLASH_ERASE
 	select DFU_TARGET_STREAM
+	# Minimal SUIT configuration to support SUIT processing
+	imply SUIT_DIGEST
+	imply SUIT_CRYPTO
+	imply SUIT_PLAT_CHECK_COMPONENT_COMPATIBILITY
+	imply SUIT_PLAT_CHECK_CLASSES
+	imply SUIT_CHECK_IMAGE_MATCH
+	imply SUIT_STREAM
+	imply SUIT_STREAM_SOURCE_CACHE
+	imply SUIT_STREAM_SOURCE_MEMPTR
+	imply SUIT_STREAM_SINK_DIGEST
+	imply SUIT_STREAM_SINK_CACHE
+	imply SUIT_STREAM_SINK_FLASH
+	imply SUIT_MEMPTR_STORAGE
+	imply SUIT_PLATFORM
+	imply SUIT_PROCESSOR
+	imply SUIT_SINK_SELECTOR
 	help
 	  Enable support for updates using DFU target based on SUIT
 
+if DFU_TARGET_SUIT
+
+config DFU_TARGET_SUIT_CACHE_PROCESSING
+	bool "SUIT cache processing"
+	imply SUIT_CACHE
```

Comment:
This is, in my opinion, wrong.

Options like SUIT_CACHE shall be selected by specific project configuration and DFU Target shall just follow.

My point is that the DFU target should not change any SUIT related configuration